packages/flutter/test/material/input_chip_test.dart

```diff
@@ -321,14 +321,14 @@ void main() {
     expectCheckmarkColor(find.byType(InputChip), Colors.white.withAlpha(0xde));
   });
 
-  testWidgets('Material3 - Input chip check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
+  testWidgets('Material3 - Input chip disabled check mark color is determined by platform brightness when dark', (WidgetTester tester) async {
     await pumpCheckmarkChip(
       tester,
       chip: selectedInputChip(),
       brightness: Brightness.dark,
     );
-
-    expectCheckmarkColor(find.byType(InputChip), Colors.white.withAlpha(0xde));
+    // Without color opacity. _RenderChip handles the 'disabled' state opacity internally
+    expectCheckmarkColor(find.byType(InputChip), ThemeData.dark().colorScheme.onSurface);
```

Comment:
```suggestion
    expectCheckmarkColor(find.byType(InputChip), theme.colorScheme.onSurface);
```

Since we're checking theme's color scheme for M3. This also needs a ThemeData variable like M3 tests.
`final ThemeData theme = ThemeData(brightness: Brightness.dark);` then pass `theme`  variable.

Reply:
Done in 0cbf9b5b9593c71902b6f58689a789004d386703
I followed the same pattern as in your other PR https://github.com/flutter/flutter/pull/140964/files